pycbc/io/hdf.py

```diff
@@ -641,12 +641,14 @@ def newsnr_sgveto_psdvar(self):
                                            self.sgchisq, self.psd_var_val)
 
     def get_column(self, cname):
-        if self.mask is not None:
+        MFRAC = 0.3
```

Comment:
please put a comment explaining what is happening here (ie why the differences). 
also, the logic would be simpler if the `self.mask is None` case was done first. 
also/however it seems like the init for the class results in self.mask *never* being None, so I'm not sure why there is still a test for `self.mask is not None` here.

Reply:
also, is there any evidence for the value of MFRAC -- i.e. what's the largest fraction of triggers for which you have successfully used the direct selection `self.trigs[cname][self.mask]` ?

Reply:
Please replace the h5py selection in `def template_id(self):` with a call to `get_column()` as for all the other datasets present in trigs.

Reply:
@tdent I'm not sure the largest MFRAC that will work, most test cases involve it being lest than 10% of the entire array. The current value is a guess.